qa/README.md

```diff
@@ -0,0 +1,27 @@
+## Acceptance test Framework
+
+Welcome to the acceptance test framework for logstash, in this small
+readme we're going to describe it's features and the the necessary steps you will need to
+follow to setup your environment.
+
+### Environment setup and Running Tests
+
+It is possible to run the full suite of the acceptance test with the codebase by 
+running the command `ci/ci_acceptance.sh`, this command will generate the artefacts, bootstrap
```

Comment:
It's spelled `artifact` when running `rake artifact:{pkgtype}`, so perhaps change the spelling here to match.